tests/kernel/pipe/pipe_api/testcase.yaml

```diff
@@ -1,3 +1,7 @@
 tests:
   kernel.pipe.api:
-      tags: kernel userspace
+    tags: kernel userspace
+  kernel.pipe.api.userspace:
```

Comment:
I am failing to see why you need this scenario with userspace enabled, all tests should be able to run on both userspace and non-userspace systems with the same code.

Reply:
This btw, will have the same test run twice the same way on platforms which enable userspace by default.

Reply:
Thanks for pointing out. I got confused before. Removed the extra test.